public/js/actions/GraphActions.js

```diff
@@ -0,0 +1,37 @@
+import dispatcher from '../dispatcher'
+import ActionTypes from '../constants/ActionTypes'
+
+export default class GraphActions {
+  static graphArticle(id) {
+    fetch(`articles/${id}`)
+            .then(res => res.json())
+            .then(article => article.keywords[0])
+            .then(word => this.updateGraph(word))
+  }
+
+  static graphTop() {
+    this.updateGraph('top')
+  }
+
+  static updateGraph(word) {
+    return fetch(`keywords/${word}`)
```

Comment:
rip that got left behind from a hack I had to test something before good catch tho.